src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menArr = century
+    ? people.filter(item => item.sex === 'm'
+      && Math.ceil(item.died / 100) === century)
+    : people.filter(item => item.sex === 'm');
```

Comment:
what if we try not to repeat code twice here?
`people.filter(item => item.sex === 'm'`